docs/blocks/blocks.md

```diff
@@ -8,18 +8,30 @@ Creating a Block
 
 ### Basic Blocks
 
-For simple blocks, which need no special functionality (think cobblestone, wooden planks, etc.), a custom class is not necessary. You can create a block by instantiating the `Block` class with a `Block.Properties` object. This `Block.Properties` object can be made using `Block.Properties#create` and it can be customised by calling its methods. For instance:
+If the block you are creating has no special functionality (think cobblestone, wooden planks), it is not necessary to create a new class for your block. You can simply instantiate the `Block` class. When doing this you will need a `Block.Properties` to pass to the constructor. This is how we set the properties of a block in 1.15+. Here's an exmaple:
```

Comment:
A list must contain at least three elements to require a comma. So either replace with 'cobblestone or wooden planks' or add the 'etc.' back in.

Small nitpick, setting properties in blocks should be an ambiguous version wise. For example, this would work on older versions as well, and as for newer versions it should be referred to via `AbstractBlock$Properties`. So avoid mentioning versions.